apps/web/src/app/wishlist/page.tsx

```
@@ -38,20 +42,16 @@ function WishlistPage() {
+  const handleConfirmDelete = async () => {
     if (selectedIds.size === 0) return;
-    setItems(prev => prev.filter(item => !selectedIds.has(item.id)));
-    setSelectedIds(new Set());
-    setIsDeleteMode(false);
-    setIsToastVisible(true);
+    try {
+      await Promise.all([...selectedIds].map(id => deleteWishlistMutation(id)));
+      setSelectedIds(new Set());
+      setIsDeleteMode(false);
+      toast.success('선택한 위시를 삭제했어요');
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)